pages/firststeps.md

```diff
@@ -25,39 +25,60 @@ A very large part of these steps is finding problems with these steps and this M
 
-##Step 3 - Markdown and Fork Tutorial:
+## Step 3 - Markdown and Fork Tutorial:
+
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/&lt;username&gt;.md).
 
-##Step 4 - BeLL-Apps Tutorial:
+## Step 4 - BeLL-Apps Tutorial:
+
 Follow the instructions under the [BeLL-Apps tab](bellapps.md).
 
-##Step 5 - Keeping Fork Updated:
+## Step 5 - Keeping Fork Updated:
+
 Follow the directions at [Git Repositories](gitandrepositories.md) to keep your username.github.io and your local repository up to date.
 
-##Step 6 - GitHub Issues Tutorial:
+## Step 6 - GitHub Issues Tutorial:
+
 Complete the tutorial under the [GitHub Issues tab](githubissues.md) and make sure to keep us posted in the Gitter chat. You should link in Gitter whenever you create an issue or when you comment on someone else's issue. Post as many issues as you can for improving the page as well as personal practice. If you know how to solve an issue, be sure to provide a detailed account of your research and show how to fix it. Next, fix the issue on your repo and create pull requests. Don't forget you can also comment and work on issues that you didn't create. The more you work and commit and push, the more impressive your GitHub profile will become. This is very important to potential employers, thus we encourage you to spend time to increase your numbers to strengthen your profile.
 
-##Step 7 - Nation BeLL:
+## Step 7 - Nation BeLL:
+
 Follow the directions under the [Nation BeLL tab](nation.md).
 
-##Step 8 - Create Issues and Pull Requests:
+## Step 8 - Create Issues and Pull Requests:
+
 Create three issues, add comments to three other issues, and resolve three issues by making the necessary changes in your repo with three separate pull requests. You will follow the same steps as in [GitHub Issues](githubissues.md), but will be working to improve this Markdown Wiki so that it is clearer and better for future use. As you have now followed along with the Markdown Wiki thus far, you should all try to improve the clarity of the explanations for these steps. Learn by teaching - or editing our teaching tool, in this case! When you are fixing an issue, mention the number of the issue you are fixing in the description of your pull request.
 
-##Step 9 - Submit Survey:
+## Before Step-9:
+
+Head over to [https://github.com/open-learning-exchange/open-learning-exchange.github.io](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and make sure you have the following done:
+
+* 5 pull requests(1 PR at step-3, 1 PR at step-6 and 3 PR's at step-8).
+* 4 comments(1 comment at step-6, 3 comment's at step-8)
+* 4 issues(1 issue at step-6, 3 issues at step-8)
+
+
+## Step 9 - Submit Survey:
 
```

Comment:
Instead of making another heading, we should just make it the first paragraph of step nine, and then make the second paragraph read "*If you meet these requirements let us know . . .*"